runtime/compiler/x/codegen/J9CodeGenerator.cpp

```diff
@@ -75,7 +75,10 @@ J9::X86::CodeGenerator::initialize()
    if (!TR::Compiler->om.canGenerateArraylets())
       {
       cg->setSupportsReferenceArrayCopy();
-      cg->setSupportsInlineStringLatin1Inflate();
+      if (feGetEnv("TR_enableStringLatin1InflateInlining"))
```

Comment:
You should evaluate `feGetEnv` into a static so that is only evaluated once per JIT process.  e.g.,
```
   static char *doSL1InfateInlining = feGetEnv("TR_enableStringLatin1InflateInlining");
   if (doSL1InflateInlining)
      {
      ...
      }
```

Reply:
Done